audbcards/core/dataset.py

```diff
@@ -390,79 +462,131 @@ def create_datacard_page(dataset: Dataset):
-                        if isinstance(mappings[0], dict):
-                            # e.g. {'s1': {'gender': 'male', 'age': 21}}
-                            mappings = sorted(list(mappings[0].keys()))
-                            mappings = f'{", ".join(mappings)}'
-                        else:
-                            # e.g. {'f': 'female', 'm': 'male'}
-                            mappings = '✓'
-                        fp.write(f', "{mappings}"')
-                fp.write('\n')
-
-
-def create_datasets_page(datasets: typing.Sequence):
-    r"""Create overview page of datasets."""
-    # Create CSV file with overview of datasets
+
+        columns = ['ID', 'Dtype']
+        header_line = '   :header: ID,Dtype'
+        if has_minimums:
+            header_line += ',Min'
+            columns.append('Min')
+        if has_maximums:
+            header_line += ',Max'
+            columns.append('Max')
+        if has_labels:
+            header_line += ',Labels'
+            columns.append('Labels')
+        if has_mappings:
+            header_line += ',Mappings'
+            columns.append('Mappings')
+
+        scheme_info['header_line'] = header_line
+        scheme_info['has_minimums'] = has_minimums
+        scheme_info['has_maximums'] = has_maximums
+        scheme_info['has_labels'] = has_labels
+        scheme_info['has_mappings'] = has_mappings
+
+        scheme_info['columns'] = columns
+        return scheme_info
+
+    @property
+    def dataset_schemes(self) -> list:
+
+        db = self.header
+        dataset_schemes = []
+        for scheme_id in db.schemes:
+            dataset_scheme = self._scheme_to_list(scheme_id)
+            dataset_schemes.append(dataset_scheme)
+
+        cols = self.scheme_info['columns']
+        data = pd.DataFrame.from_dict(dataset_schemes)[cols]
+        filter = data.applymap(lambda d: d == [])
+        data.mask(filter, other='', inplace=True)
+        scheme_data = np.array(data).tolist()
+        return scheme_data
+
+
+class Datacard(object):
+
+    def __init__(self, dataset: Dataset):
+
+        self._dataset = dataset
+
+    @property
+    def props(self) -> dict:
+        """slot to hold superclass properties for rendering."""
+        props = self._dataset.properties()
+        return props
+
+    @functools.cached_property
+    def content(self):
+        """Property Accessor for rendered jinja2 content."""
+
+        return self._render_template()
+
+    def _render_template(self):
+
+        t_dir = os.path.join(os.path.dirname(__file__), 'templates')
+        environment = jinja2.Environment(loader=jinja2.FileSystemLoader(t_dir),
+                                         trim_blocks=True)
+        # Provide Jinja filter access to Python build-ins/functions
+        environment.filters.update(
+            zip=zip,
+            tw=self._trim_trailing_whitespace,
+        )
+        template = environment.get_template("datacard.j2")
+        content = template.render(self.props)
```

Comment:
I would not add a public property for this, but simply call the required property here directly:

```suggestion
        content = template.render(self._dataset.properties())
```